### PR TITLE
upgrade tifffile

### PR DESCRIPTION
**Pull request by benjijamorris**
## What does this PR do?
Previous tests were failing because the tifffile version in the test-requirements was too old. Regular installation worked, but for the sake of passing tests, this enforces the tifffile version. 

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **test your PR locally** with `pytest` command?
- [x] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by pgarrison** [comment]
Great job figuring this out! Tentatively seems good, based on the logs of the Test job, but let's merge the result of [this action](https://github.com/AllenCellModeling/cyto-dl/actions/runs/11242934313) into this feature branch before going ahead, to confirm the bug is fully resolved.

File: requirements/linux/all-requirements.txt [hunk 1]
```diff
@@ -2,23 +2,23 @@
 # Please do not edit it manually.
 
 aicscytoparam==0.1.12
-aicsimageio==4.14.0
+aicsimageio==4.11.0
```
Comment (ritvikvasan) [inline requirements/linux/all-requirements.txt:5]: tifffile was upgraded but aicsimageio was downgraded? Also do we install both bioio and aicsimageio?

**Review by ritvikvasan — APPROVED** [review]
How does aicsimageio/bioio installation work now? Do we just use bioio?

File: requirements/linux/all-requirements.txt (hunk 1 above)
Comment (benjijamorris) [reply requirements/linux/all-requirements.txt:5]: cyto-dl just uses bioio, aicsimageio is a dependency of pyshtools

**Pull request merged**